packages/amplication-client/src/util/analytics.ts

```diff
@@ -5,6 +5,8 @@ import { version } from "../util/version";
 
 export interface Event {
   eventName: AnalyticsEventNames;
+  /** The location of a button / HTML element. i.e. WorkspaceHeader */
+  eventOriginLocation?: string;
```

Comment:
I wouldn't add that to the interface, we have a lot of properties that repeat themself in that event. either you'll add them all or leave it dynamic to event